### DISCUSSION TITLE
Regenerate supabase api keys

### DISCUSSION THREAD

**Discussion by misante** [supabase/supabase#30399] [Questions]
How to regenerate supabase api keys?

**Comment by GaryAustin1** [comment]
You reset the jwt secret currently.

**Reply by GaryAustin1** [reply]
Make sure to read the warning when you go to change it...
![image](https://github.com/user-attachments/assets/c22b4d55-e6e8-4163-8f92-def77b02cd70)

This is supposed to change later this year.

**Reply by misante** [reply]
> You reset the jwt secret currently.

Do You mean, resetting jwt secret, will automatically regenerate anon and service api?

**Reply by GaryAustin1** [reply]
Yes
